### PR TITLE
PT-579 Integrate tidepool v0.8.0

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,35 +10,36 @@
...
+    "hakken": "0.1.12",
+    "lodash": "4.17.11",
+    "moment": "^2.24.0",
+    "mongojs": "2.6.0",
+    "request": "2.88.0",
+    "restify": "8.3.2",
+    "sundial": "1.6.0",
+    "tidepool-gatekeeper": "0.2.4",
+    "tidepool-seagull-client": "0.1.8",
+    "user-api-client": "mdblp/user-api-client#feature/pt583-v0.5.0"
```
Comment (CedricPerotto) [inline package.json:31]: We don't use our forked version. 
```suggestion
    "user-api-client": "0.5.0"
```

File: version.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
-export ARTIFACT_NODE_VERSION="6.10.2"
-export START_NODE_VERSION="${ARTIFACT_NODE_VERSION}"
\ No newline at end of file
+export ARTIFACT_NODE_VERSION="10.15.3"
```
Comment (CedricPerotto) [inline version.sh:1]: The node version upgrade is out of scope for this PR
```suggestion
export ARTIFACT_NODE_VERSION="10.14.2"
```

File: .travis.yml [hunk 3]
```diff
@@ -2,19 +2,15 @@ sudo: false
 
 language: node_js
 
+dist: bionic
+
 node_js:
-  - 6.10.2
+  - 10.15.3
```
Comment (CedricPerotto) [inline .travis.yml:8]: The node version upgrade is out of scope for this PR

```suggestion
  - 10.14.2
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 Tidepools message API for adding context to diabetes data
 
+## 0.2.0
+### Changed
+- [PT-579] Integrate Tidepool [v0.8.0](https://github.com/tidepool-org/message-api/releases/tag/v0.8.0)
+- Upgrade node to 10.15.3
```
Comment (CedricPerotto) [inline CHANGELOG.md:8]:
```suggestion
```

File: Dockerfile [hunk 5]
```diff
@@ -1,4 +1,4 @@
-FROM node:6.10.2-alpine
+FROM node:10.15-alpine
```
Comment (CedricPerotto) [inline Dockerfile:1]: The node version upgrade is out of scope for this PR
```suggestion
FROM node:10.14.2-alpine
```

File: Dockerfile [hunk 6]
```diff
@@ -7,14 +7,12 @@ WORKDIR /app
 
 COPY package.json .
 
-RUN apk add --no-cache --virtual .build-dependencies git && \
-    sed -i -e 's/"mongojs": "0.18.2"/"mongojs": "2.4.0"/g' package.json && \
-    yarn install && \
-    yarn cache clean && \
+RUN apk add --no-cache --virtual .build-dependencies git python make && \
```
Comment (CedricPerotto) [inline Dockerfile:10]: python & make packages are not used (we can build and run the container without them) 
```suggestion
RUN apk add --no-cache --virtual .build-dependencies git && \
```

File: Dockerfile [hunk 7]
```diff
@@ -7,14 +7,12 @@ WORKDIR /app
...
+    npm install && \
+    npm cache clean --force && \
     apk del .build-dependencies
 
 USER node
 
 COPY . .
-
-CMD ["npm", "start"]
+CMD ["node", "lib/index.js"]
```
Comment (CedricPerotto) [inline Dockerfile:18]: I don't understand the purpose of this change (absent from tidepool merge)  
```suggestion
CMD ["npm", "start"]
```

**Review by CedricPerotto — CHANGES_REQUESTED** [review]

**Comment by yvon-dblg** [comment]
Fix on https://github.com/mdblp/message-api/pull/10

**Pull request closed without merging**